app.json

```diff
@@ -11,7 +11,7 @@
     ],
     "version": "1.0.0",
     "orientation": "portrait",
-    "icon": "./assets/icon.png",
```

Comment:
@snowmanstark i dont really understand why we need to do this.

Reply:
Due to errors saying the icon size is 192x110.

Reply:
you should try `expo optimize` it should resolve your issue.

Reply:
Even after `expo optimize` the console shows the error mesage `Error: Problems validating asset fields in app.json. See https://docs.expo.io/
 • Field: icon - image should be square, but the file at './assets/icon.png' has dimensions 192x110.`